frontend/www/js/omegaup/components/common/MultiUserAddArea.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <div class="multi-user-add-area">
+    <textarea
+      v-if="isFocused || !usersList.length"
+      v-model="bulkContestants"
+      class="form-control contestants"
+      data-contestant-names
+      @input.prevent="onBulkContestantsChanged"
+      @blur.prevent="isFocused = false"
+    >
+    </textarea>
+    <div v-else class="form-control contestants">
+      <span href="#" class="edit-icon" @click.prevent="isFocused = true"
+        >Editar</span
+      >
+      <div class="users-list">
+        <div v-for="user in usersList" :key="user" class="users-list__item">
+          <span
+            class="tags-input-badge tags-input-badge-pill tags-input-badge-selected-default"
+            >{{ user }}</span
+          >
+          <a
+            href="#"
+            class="tags-input-remove"
+            @click.prevent="removeUser(user)"
+          ></a>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+const debounce = (fn: (event: Event) => void, waitTime: number) => {
+  let timer: any = null;
+
+  return (...args: any) => {
+    if (timer) {
+      clearTimeout(timer);
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
+    const context = this;
+
+    timer = setTimeout(() => {
+      fn.apply(context, args);
+    }, waitTime);
+  };
+};
+
+const WAIT_TIME = 1000;
+import { Vue, Component, Watch, Prop } from 'vue-property-decorator';
+
+@Component({})
+export default class MultiUserAddArea extends Vue {
+  @Prop() users!: string[];
+  isFocused: boolean = false;
+  bulkContestants: string | null = '';
```

Comment:
Si este dato ya permite valores nulos, ¿qué te parece si le asignamos `null` de inicio?

```suggestion
  bulkContestants: string | null = null;
```